fortio_main.go

```diff
@@ -32,6 +32,12 @@ import (
 	"istio.io/fortio/ui"
 )
 
+var httpOpts fhttp.HTTPOptions
+
+func init() {
+	httpOpts.Init("")
```

Comment:
any reason to prefer this over: `httpOpts = NewHTTPOptions("")` ?

Reply:
it avoids a copy, initializes in place the existing struct. but I could I guess make httpOpts a pointer